Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = [
+    "bee-api/bee-rest-api",
```

Comment:
align

Reply:
that sounds somewhat authoritarian, Thibault :trollface: